arcgis-ios-sdk-samples/Route and directions/Display route layer/DisplayRouteLayer.storyboard

```diff
@@ -0,0 +1,94 @@
+                        <constraints>
+                            <constraint firstItem="WUb-UN-MJ9" firstAttribute="leading" secondItem="5EZ-qb-Rvc" secondAttribute="leading" id="AfZ-ta-JlV"/>
+                            <constraint firstItem="etS-o9-hpR" firstAttribute="leading" secondItem="5EZ-qb-Rvc" secondAttribute="leading" id="Fc6-1q-ayj"/>
+                            <constraint firstItem="WUb-UN-MJ9" firstAttribute="top" secondItem="etS-o9-hpR" secondAttribute="bottom" id="Zlv-NN-hfp"/>
+                            <constraint firstAttribute="trailing" secondItem="WUb-UN-MJ9" secondAttribute="trailing" id="kUl-U2-DqK"/>
+                            <constraint firstItem="etS-o9-hpR" firstAttribute="top" secondItem="5EZ-qb-Rvc" secondAttribute="top" id="mBu-AT-wK5"/>
+                            <constraint firstItem="vDu-zF-Fre" firstAttribute="bottom" secondItem="WUb-UN-MJ9" secondAttribute="bottom" id="mbX-DF-UWa"/>
+                            <constraint firstAttribute="trailing" secondItem="etS-o9-hpR" secondAttribute="trailing" id="vLn-84-qaX"/>
+                        </constraints>
+                    </view>
+                    <connections>
+                        <outlet property="directionsBarButtonItem" destination="65F-7g-wNV" id="tPV-OW-VgX"/>
+                        <outlet property="mapView" destination="etS-o9-hpR" id="hbp-fI-IlH"/>
+                    </connections>
+                </viewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="Ief-a0-LHa" userLabel="First Responder" customClass="UIResponder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="139" y="54"/>
+        </scene>
+        <!--Display Directions View Controller-->
+        <scene sceneID="Iqd-Rc-dQG">
+            <objects>
+                <tableViewController id="Shx-Rl-xgK" customClass="DisplayDirectionsViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" allowsSelection="NO" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="-1" estimatedSectionHeaderHeight="-1" sectionFooterHeight="-1" estimatedSectionFooterHeight="-1" id="bzd-j8-nNX">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="842"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <color key="backgroundColor" systemColor="systemBackgroundColor"/>
+                        <prototypes>
+                            <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" selectionStyle="default" indentationWidth="10" reuseIdentifier="DirectionsCell" id="njd-w9-Ll4">
```

Comment:
Do you think it's OK to use the basic style cell?

<img width="296" alt="image" src="https://user-images.githubusercontent.com/9660181/167524435-8a439142-16a0-46ec-8e36-161f250b0fc3.png">